content/02.intro.md

```diff
@@ -1 +1,10 @@
 ## Introduction
+
+Deep learning (DL), a subfield of machine learning (ML), is exploding in popularity and is increasingly being used for biological data analysis [@doi:10.1098/rsif.2017.0387].
+However, DL is large and complex field, and its correct application to biological research remains a daunting task for most.
```

Comment:
A suggested alternative: 

However, DL itself remains and active area of research, and the complexity of this field poses a large barrier of entry to those who would like to utilize state-of-the-art DL technology in biological research applications.